tests/ballerina-unit-test/src/test/resources/test-src/expressions/stamp/anydata-stamp-expr-test.bal

```diff
@@ -0,0 +1,93 @@
+type Employee record {
+    string name;
+    string status;
+    string batch;
+};
+
+type Teacher record {
+    string name;
+    int age;
+    string status;
+    string batch;
+    string school;
+};
+
+function stampAnydataToJSON() returns json|error  {
+
+    anydata anydataValue = 3;
+    json|error  jsonValue = json.stamp(anydataValue);
+
+    return jsonValue;
+}
+
+function stampAnydataToRecord() returns Employee|error  {
+    Teacher t1 = { name: "Raja", age: 25, status: "single", batch: "LK2014", school: "Hindu College" };
+    anydata anydataValue = t1;
+    Employee|error  employee = Employee.stamp(anydataValue);
+    return employee;
+}
+
+function stampAnydataToJSONV2() returns json|error  {
+    json t1 = { name: "Raja", age: 25, status: "single", batch: "LK2014", school: "Hindu College" };
+    anydata anydataValue = t1;
+
+    json|error  jsonValue = json.stamp(anydataValue);
+    return jsonValue;
+}
+
+function stampAnydataToXML() returns xml|error  {
+
```

Comment:
Shall we remove the extra line added after each function signature?

Reply:
Fixed.